Adding relation options description

frankorama opened the pull request:
The 'options' property was missing form the definition of the Relations property.  Adding it to the v3 documentation

frankorama committed:
Adding relation options description

The 'options' property was missing form the definition of the Relations property.  Adding it to the v3 documentation

slnode commented:
Can one of the admins verify this patch?

crandmck commented:
Please see https://github.com/strongloop/loopback.io/pull/399#issuecomment-308845141

frankorama committed:
options.disableInclude: removed table in table

The description of the relations property now has options.disableInclude as a single item instead of defining the options as a table within the relations properties description table.

crandmck commented:
LGTM, thank you!
